### DISCUSSION TITLE
Copilot in Intellij Idea for maintainer of a popular repository

### DISCUSSION THREAD

**Discussion by vdiravka** [community/community#31698] [Copilot Conversations]
* I have write access to the popular repository ([Apache Drill](https://github.com/apache/drill)), but I don't see free subscription here:
https://github.com/github-copilot/signup
![image](https://user-images.githubusercontent.com/11904420/188070081-373a2579-1eb6-43c1-ace6-991ed3252957.png)
_Note:_ this is similar to https://github.com/orgs/community/discussions/30424, but I am going to apply for CoPilot usage at first.

* The other issue (possibly for another ticket), Intellij Idea not signed in to GitHub even after finishing successful log in process (see pics):
![Screenshot from 2022-09-02 09-09-29](https://user-images.githubusercontent.com/11904420/188070358-82e3fdd8-c417-4bbf-a1bb-c6a4bf1407c9.png) 
->
![image](https://user-images.githubusercontent.com/11904420/188070332-2fe49e12-472c-4c7d-9efd-72041226e16a.png) 
->
![image](https://user-images.githubusercontent.com/11904420/188070513-d0400e36-1aff-437e-80dc-e98bb51d7f84.png)

_Note:_ I don't have error message after the process like in https://github.com/orgs/community/discussions/29243. The login window just disappeared in the Intellij Idea

**Comment by zcong1993** [comment]
I have the same problem, did you get free subscription for now?

**Reply by vdiravka** [reply]
No, Copilot team didn't help me yet :(

**Comment by matthiastjong** [comment]
Same problem here ...
